litmus-portal/frontend/src/components/DateTime/CustomDate/index.tsx

```diff
@@ -26,17 +26,20 @@ const CustomDate: React.FC<CustomDateProps> = ({ disabled }) => {
       <KeyboardDatePicker
         className={classes.datePicker}
         autoOk
-        variant="inline"
         format="dd/MM/yyyy"
         id="date-picker-inline"
         inputVariant="outlined"
         disabled={disabled}
+        cancelLabel="Cancel"
+        okLabel="Save"
+        variant="dialog"
         value={selectedDate}
         inputProps={{
           style: {
-            fontSize: '0.8125rem',
+            fontSize: '0.75rem',
             color: '#000000',
```

Comment:
Can you use theme here? It will look good. Also use CSS in styles.ts.

Reply:
Actually it is in the input props, we can't change that